lib/models/application.coffee

```diff
@@ -278,17 +278,10 @@ getApplicationModel = (deps, opts) ->
 			options:
 				mergePineOptions
 					$filter:
-						$eq: [
-							$tolower: $: 'app_name'
-							appName
-						],
-						user:
-							$any:
-								$alias: 'u',
-								$expr: $eq: [
-									$tolower: $: 'username'
-									owner
-								]
+						$or: [
+							{ slug: "#{owner}/#{appName}" }
+							{ slug: "#{appName}" }
```

Comment:
I think we should only have the first case here. I think there's a good argument that calling `getAppByOwner` in open-source (or using abc/appname in the CLI) should return nothing. You're filtering by owner, and the apps don't have the owner you're asking for (because they have no owner at all).